### DISCUSSION TITLE
Using cypress.env.json secrets with act

### DISCUSSION THREAD

**Discussion by 9j0hn50n** [nektos/act#1762] [Q&A]
Hi, I have an uncommitted `cypress.env.json` file that contains secrets used as part of my Cypress e2e tests.

These are referenced in my GitHub workflow for usage in CI.

When I run act, I get a failure because the mentioned Cypress secrets can't be found.

Do I pass in `cypress.env.json` using the `--secret-file` flag? If so, can I pass multiple files as I'm already using `--secret-file .env`.

But, I don't think this will work as the readme says the format should be the same as a `.env` file.

If I cant pass `cypress.env.json` in how do I go about using my Cypress secrets as part of act?

Many thanks!

**Comment by ChristopherHX** [comment]
If you use act 0.2.45 (1. May 2023) or a build from master. You can rename `cypress.env.json` to `cypress.env.yml` and use it as a yml (most json is also valid yaml) secret file.
> can I pass multiple files

No, this is not implemented in act.
You would still need to merge both secret files into one format if you need both.
